index.html

```diff
@@ -8,38 +8,68 @@
 </head>
 
 <body>
-    <center>
-        <h1 style="margin-top: 20px; margin-bottom: 20px; ">
-           All awesome books
-        </h1>
-    <div class="displayWrap">
-        <div id="display_card" style="font-weight: bold;">
+    <header>
+        <nav class="nav">
+            <h1>All Awesome Books</h1>
+            <ul>
+                <li class="menuList"><a href="#">List</a></li>
+                <span style="display: inline-block; width:3px; background-color:rgb(0, 0, 0); color: black;"></span>
+                <li class="menuAdd"><a href="#">Add New</a></li>
+                <span style="display: inline-block; width:3px; background-color:rgb(0, 0, 0); color: black;"></span>
+                <li class="menuContact"><a href="#">Contact</a></li>
+            </ul>
+        </nav>  
+    </header>
 
-        </div>
-    </div>
-   <hr>
-    <h2 style="margin-top: 20px; margin-bottom: 20px; ">
-        Add a new book
-     </h2>
+    <!-- Books List -->
+    <Section class="sectionList">
+        <center>
```

Comment:
- [ ] The `<center>` element is deprecated because it defines the presentation of its contents — it does not describe its contents. To center your element I would like to recommend you use the margin-left and margin-right properties of the element to auto, and then set the parent element's text-align property to center. Please fix it because it is a good practice to not use deprecated element for security reason :+1: